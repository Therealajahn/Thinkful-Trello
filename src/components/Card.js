import React from 'react';




function Card(props) {
  
  return (
    <div className="Card">
            <button type="button" onClick={() => {props.deleteCard(props.listId, props.cardId)}}>delete</button>
            <h3>{ props.title }</h3>
            <p>{ props.content }</p>
    </div>
  );
}

export default Card;