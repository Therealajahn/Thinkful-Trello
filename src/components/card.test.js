import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Card from './Card.js';

describe( 'card', () => {
    it('renders without crashing',() =>{
        const container = document.createElement('div');
        ReactDOM.render(
            <Card />,
            container
        );
        ReactDOM.unmountComponentAtNode(
           container
        )
    })
    it("card hasn't changed", () =>{
        const tree = renderer.create(<Card />).toJSON();
        expect(tree).toMatchSnapshot();
    })
}
    
)


