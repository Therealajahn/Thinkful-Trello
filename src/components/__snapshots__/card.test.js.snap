// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`card card hasn't changed 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3 />
  <p />
</div>
`;
